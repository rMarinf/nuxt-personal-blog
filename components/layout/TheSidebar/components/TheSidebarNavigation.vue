<template>
  <nav id="side-menu-main" role="navigation">
    <ul>
      <li class="active"><a href="travel.html">Blog</a></li>
      <li><a href="lifestyle.html">Sobre m√≠</a></li>
      <li><a href="about.html">Contacto</a></li>
    </ul>
  </nav>
</template>

<script>
export default {
  name: 'TheSidebarNavigation'
}
</script>

<style scoped></style>
