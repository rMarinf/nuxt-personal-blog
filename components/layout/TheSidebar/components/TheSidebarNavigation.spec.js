import { shallowMount } from '@vue/test-utils'
import TheSidebarNavigation from './TheSidebarNavigation.vue'

test('TheSidebarNavigation', () => {
  // render the component
  const wrapper = shallowMount(TheSidebarNavigation)

  expect(wrapper.element).toMatchSnapshot()
})
