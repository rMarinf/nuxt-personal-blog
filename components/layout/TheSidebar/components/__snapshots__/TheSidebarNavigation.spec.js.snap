// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSidebarNavigation 1`] = `
<nav
  id="side-menu-main"
  role="navigation"
>
  <ul>
    <li>
      <nuxt-link-stub
        to="/"
      >
        Blog
      </nuxt-link-stub>
    </li>
     
    <li>
      <nuxt-link-stub
        to="/about"
      >
        Sobre m√≠
      </nuxt-link-stub>
    </li>
     
    <li>
      <nuxt-link-stub
        to="/contact"
      >
        Contacto
      </nuxt-link-stub>
    </li>
  </ul>
</nav>
`;
